source/xrefdelta.tex

```diff
@@ -106,5 +106,9 @@
 % LWG3659 Consider ATOMIC_FLAG_INIT undeprecation
 \removedxref{depr.atomics.flag}
 
+% LWG3818 Exposition-only concepts are not described in library intro
+\movedxref{expos.only.func}{expos.only.entity}
+\removedxref{expos.only.types}
```

Comment:
Why not `\movedxref`? It is removed, but the specification was merged to [expos.only.entity].

Reply:
The section itself (name and number) no longer exist, hence `\removedxref`. `\movedxref` is for sections that are renamed. We don't have a way of documenting that a section contents were merged into another section afaik, we just move the contents.

Reply:
FYI, this is from LWG3818 which was not intended to be part of the paper.